addons/payment/wizards/payment_capture_wizard.py

```diff
@@ -87,7 +87,9 @@ def _compute_is_amount_to_capture_valid(self):
     def _compute_support_partial_capture(self):
         for wizard in self:
             wizard.support_partial_capture = all(
-                tx.provider_id.support_manual_capture == 'partial' for tx in wizard.transaction_ids
+                tx.provider_id.support_manual_capture == 'partial' \
+                and tx.payment_method_id.support_manual_capture == 'partial'\
```

Comment:
`\` are not necessary inside parenthesis.
```suggestion
                tx.provider_id.support_manual_capture == 'partial'
                and tx.payment_method_id.support_manual_capture == 'partial'
```